elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -280,12 +280,24 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                         continue
 
                     # Get corresponding property's value from parsed pipeline
-                    property_value = operation.component_params.get(component_property.ref)
+                    property_value_dict = operation.component_params.get(component_property.ref)
+                    property_value = property_value_dict[property_value_dict['activeControl']]
+
+                    self.log.debug(f"Property value : {property_value}")
 
                     self.log.debug(f"Processing component parameter '{component_property.name}' "
                                    f"of type '{component_property.data_type}'")
 
-                    if component_property.data_type == "string":
+                    if property_value and not type(property_value) == bool and \
+                            str(property_value)[0] == '{' and str(property_value)[-1] == '}':
```

Comment:
Can `property_value` be `None` or the empty string?  If so, is there a way to prevent that case so this condition (the `property_value` has a value portion) can be removed?

Reply:
It just occurred to me that even if `property_value` was of `type` `bool`, that portion of the clause still wouldn't have any impact on the checks for the begin/end curly braces, so is that clause necessary?  If the value was a boolean, it would be an incantation of True or False (1 or 0, etc.) none of which would include curly braces.